examples/eks/iam.tf

```diff
@@ -0,0 +1,113 @@
+# IAM user required for CAST.AI
+
+provider "castai" {
+  api_url   = "https://console.cast.ai//"
+  api_token = var.castai_api_token
+}
+
+provider "aws" {
+  region  = var.region
+}
+
+locals {
+  iam_user = "castai-eks-${var.cluster_name}"
+  aws_account_id = data.aws_caller_identity.current.account_id
+  instance_profile = "cast-${substr(var.cluster_name,0,40)}-eks-${substr(var.cluster_id,0,8)}"
+  instance_profile_roles = toset([
```

Comment:
Worth extracting the roles lists into the same datasource

Reply:
agreed